runtime/optwin.vim

```diff
@@ -810,6 +810,12 @@ call <SID>OptionL("ts")
 call append("$", "shiftwidth\tnumber of spaces used for each step of (auto)indent")
 call append("$", "\t(local to buffer)")
 call <SID>OptionL("sw")
+call append("$", "vartabstop\tlist of number of spaces a tab counts for")
```

Comment:
Do not remove feature checks. Most files in `runtime/` is maintained via upstream patches so avoid diverging from upstream unless it is necessary.